### ISSUE TITLE
Bug in AutocompletePageTest

### ISSUE THREAD

**Issue by warpech** [StarcounterSamples/KitchenSink#50]
There is a potential bug on line https://github.com/StarcounterSamples/KitchenSink/blob/7b12ed2a1d681dd747ffdeb30ff224c65fbbb935/test/KitchenSink.Tests/Tests/AutocompletePageTest.cs#L50.

Which causes the test to sometimes fail. Are we sure that `Assure.IsEmpty` will always be empty after clicking?

**Comment by warpech** [comment]
cc @joozek78, @alemoi

**Comment by joozek78** [comment]
Well, it definitely should be. It probably fails because of race condition (waiting for puppet is missing).

**Comment by alemoi** [comment]
I agree with you guys. I experienced very occasional behaviour(success/failure) on the same machine.

**Comment by warpech** [comment]
I guess that the error is between these lines:

```cs
driver.FindElements(FoundPlacesSelector)[0].Click();
Assert.IsEmpty(driver.FindElements(FoundPlacesSelector));
```

After the test clicks on an element, there is a roundtrip to server to clear the list. So we should wait for the list to be cleared, because it can take few milliseconds.

**Comment by joozek78** [comment]
Closed by #49
